src/client/utils/property-overriding.ts

```diff
@@ -54,3 +55,53 @@ export function overrideDescriptor<O extends object, K extends keyof O> (obj: O,
 
     nativeMethods.objectDefineProperty(obj, prop, descriptor);
 }
+
+function overrideFunctionName (fn: Function, name: string) {
+    const wrapperNameDescriptor = nativeMethods.objectGetOwnPropertyDescriptor(fn, 'name');
```

Comment:
`const nameDescriptor =...`

Reply:
Use `//eslint-disable-line` because it makes the code more readable.
Use `fast forward` strategy here to avoid additional brackets:
```js
 if (!nameDescriptor)
    return;

nameDescriptor.value = name; // eslint-disable-line no-restricted-properties
```